### DISCUSSION TITLE
[Feature]  Sharing albums across multiple Immich instances

### DISCUSSION THREAD

**Discussion by axgd-code** [immich-app/immich#7039] [Feature Request]
### The feature

The proposed feature entails the ability to share albums between different instances of Immich. The primary objectives are:
- Sharing albums with users outside of one's own instance.
- Allowing users to import or display remote albums within their own instances.
- Facilitating easier photo importing for users from other instances.
- Implementing multi-owner albums across multiple instances.
- This feature is envisioned as the missing functionality to replace Google Photos' "Shared with Me" feature.

Technical Considerations:
Without delving into specific technical solutions, implementing a sharing solution via the ActivityPub standard could greatly benefit Immich. ActivityPub is an open standard that fosters interoperability between different social networking platforms. Integrating this standard could streamline the sharing process and enhance the platform's overall functionality.

Linked issues:
- [2730](https://github.com/immich-app/immich/discussions/2730)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #728, #916

**Reply by axgd-code** [reply]
Thank you for these links @bo0tzz :). 
I'm rather concerned if this topic isn't on the roadmap, as the "sharing"/"social" aspect of Google Photos won't be equivalent and may drive away potential users. If I manage to find some free time, I'd be happy to get involved in a technical proposal.
